sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md

```diff
@@ -3,17 +3,15 @@
 ## 1.3.0-beta.2 (Unreleased)
 
 ### Features Added
+- Added configuration settings snapshot feature which allow users to create a point-in-time snapshot of their configuration store.
 
 ### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Removed protocol methods from the `ConfigurationClient`.
```

Comment:
We generally want to avoid listing out breaking changes between betas, since that is an expected experience and would be a large volume in most change logs, making it harder for customers to identify any breaking changes to GA releases.

Let's shift this to "Other changes" so that we provide the awareness.